docs/gitrepo-add.md

```diff
@@ -128,3 +128,31 @@ stringData:
 
 Note: If you don't add it any server's public key will be trusted and added.
 
+# Troubleshooting
+
+### Fleet deployment stuck in modified state
+
+When you add a Helm chart to Fleet, some of the components are modified, and this causes the "modified" flag in the Fleet environment.
+
+To ignore the modified flag for the differences between the Helm install generated by `fleet.yaml` and the resource in your cluster, add a `diff.comparePatches` to the `fleet.yaml` for your Deployment, as shown in this example:
```

Comment:
If you specifically reference a Deployment resource, you should comparePatch a Deployment resource in the example yaml below such as https://github.com/ibrokethecloud/core-bundles/blob/21f9ab03f89b52ff3b60c52e1cfedcee7e23e8b0/monitoring/fleet.yaml#L45-51

Reply:
I agree with the analysis. Do you have a suggested change here?

Reply:
Changed code sample to:


```yaml
defaultNamespace: <namespace name> 
helm:  
  releaseName: <release name>  
  repo: <repo name> 
  chart: <chart name>
diff:  
  comparePatches:  
  - apiVersion: apps/v1Gaurav Mehta, 4 months ago: • Working Repo with common base charts
    kind: Deployment
    operations:
    - {"op":"remove", "path":"/spec/template/spec/hostNetwork"}
    - {"op":"remove", "path":"/spec/template/spec/nodeSelector"}
    - {"op":"remove", "path":"/spec/template/spec/priorityClassName"}
    - {"op":"remove", "path":"/spec/template/spec/tolerations"}
```